CMSgov/eAPD
# Tbolt/3355 budget and ffp validation

Resolves #3355 

### Description
Adds validation to the Federal-State Split and Estimated Quarterly Expenditure sections of the Budget & FFP page

### Significant changes or possible side effects

### Automated test cases written

### Steps to manually verify this change

1. Login and view an APD
2. Verify that Federal-State Split shows an error when an option is not selected
3. Verify that the Estimated Quarterly Expenditure table rows must equal 100% or an error is shown
Note: for testing we have the admin check turned on by default with no way to turn it off. So the testing should be done with that in mind.

### This pull request is ready to review when


### This pull request can be merged when
